tetrateci/tsb-util/manifests/gateway.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - hosts:
+    - bookinfo5.k8s.local
```

Comment:
Why is this hostname hardcoded - this should be dynamically generated

Reply:
It is generated dynamically this is just a placeholder